### DISCUSSION TITLE
I've been having a hard time: I'll take a break and we'll extend by 2 weeks

### DISCUSSION THREAD

**Discussion by kaspth** [kaspth/oaken#51] [Announcements]
Hey everybody, I've been having a hard time lately and I've been finding it hard to get ideas for or work on Oaken.

I tend to feel a lot of pressure when I do stuff like this because I want to be great — but I often forget that I'm a person and need to take it easy too. And then the pressure just grows and I get less done and less ideas, and so on and so on. It usually takes me a while to admit it to myself too. But here we are! It's clear to me that I need to give myself a conscious break from thinking about Oaken, it's been draining me in the background.

So I'm taking this week explicitly off and then we'll extend this until Nov 14 instead of Oct 31. Maybe I'll need another week after that. And I'll try not to worry that I'll keep extending it (I'm also not the best at trusting myself, so…).

------

If you've been finding this useful at all, I'd love to hear from you as a comment here and what you've been getting out of this so far. You're welcome to share ideas for what you'd like to see too. I'm not the best with criticism right now, so if you could hold that right now I'd appreciate it 😅

Since we're halfway through, I've dropped the prices a bit — so if you feel like sharing https://kaspthrb.gumroad.com/l/open-source-retreat-summer-2023 on social media, that'd help me out a lot too.

Thank you for your participation in this! I hope it's ok that I show that I haven't got it all figured out, and sometimes working on code gets difficult for me too.

**Comment by larkro** [comment]
I totally relate to it all! The feeling of pressure, understand how ambition drains, and full-hearted support some time off! <3 
Love the openness, about it all, from specific technical difficulties to struggles in boundaries and more. Thank you for sharing.

For me, this thing is great! :-) I learn a lot, get inspired and some of the motivation/joy of programming is slowly coming back. <3 I'm starting to dream again about my pet project and really feel that Oaken would help me build it. Thank you very much for everything so far. I'm sorry that I haven't been a better contributor, or moral supporter. I will reflect about what I can do on my part. You are not alone. All the best.

**Comment by codergeek121** [comment]
Even though I have not been able to catch up with everything lately, this has been a great learning resource for me so far. I think it's my favorite format, just asynchronously reading advanced code 👍Don't worry about weekly output, I much prefer quality over quantity.

Also if anyone here is at RailsWorld and wants to meet and discuss some ideas for Oaken or pair on some ideas, I'd be happy to meet and chat 👍

**Comment by salochara** [comment]
Totally understand @kaspth !

Take as much time as you need. All the best!!
